README.md

```diff
@@ -1 +1,183 @@
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JAVASCRIPT</li>
+    <li>BOOTSTRAPL</li>
+  </ul>
+</details>
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Responsive design**
+- **Used only Bootstrap, CSS, html and Javascript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](https://hanasabih.github.io/awesome-books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- A personal computer connected to the internet
+- A Modern/Updated web browser
+- Have a local version control like git installed on your computer and a Github account
+- Code editor installed on your computer
+
+
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```sh
+    cd your-desired-folder
+    git clone https://github.com/HanaSabih/awesome-books.git
+```
+
+
+### Install
+
+Install this project with:
+```sh
+    cd your-desired-folder
+    git clone https://github.com/HanaSabih/awesome-books.git
+```
+
+
+
+
+### Usage
+
+To run the project, open index.html with your preferred browser.
+
+
+
+### Run tests
+
+To run tests, run the following command in your `git bash` or CLI:
+- `npx hint .` : for the html codebase and use of best practices.
+- `npx stylelint "**/*.{css,scss}" `: for the css codebase and use of best practices. you can add.
+- `npx eslint . `: for the JavaScript codebase and use of best practices.
+
+
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Hana Sabih**
+
+- GitHub: [@githubhandle](https://github.com/HanaSabih)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/hana-sabih/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Program Page**
+- [ ] **Register Page**
+- [ ] **News Page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, kindly give a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] OPTIONAL
It's recommended as good practice that you add an MIT license or any other similar license to your ReadMe for this project. Kindly endeavor to improve upon this aspect

Reply:
thank you for your recommendation.